Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -3722,9 +3724,31 @@ IDropDownMenu IBocList.OptionsMenu
       }
     }
 
+    /// <inheritdoc />
+    public WebString ListMenuHeading
+    {
+      get => _listMenuHeading;
+      set => _listMenuHeading = value;
+    }
+
+    /// <inheritdoc />
+    public HeadingLevel? ListMenuHeadingLevel
```

Comment:
```suggestion
    /// <inheritdoc />
    [Category("Menu")]
    [Description("The heading level applied to the ListMenuHeading. Leave empty to use a SPAN instead of a Heading-element.")]
    [DefaultValue(typeof(HeadingLevel?), "")]
    public HeadingLevel? ListMenuHeadingLevel
```